app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java

```diff
@@ -78,10 +86,30 @@ public static void deleteAll(Context c) {
 		return measurements;
 	}
 
+	/*
+	 * Sorting measurements:
+	 * by is_anomaly and is_failed for Websites
+	 * Whatsapp - Telegram - Facebook for Instant Messaging
+	 * Ndt - Dash - HIRL - HHFM for Performance
+	 * Psiphon - Tor for Circumvention
+	 */
 	public List<Measurement> getMeasurementsSorted() {
-		if (measurements == null)
-			measurements = SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true)).orderBy(Measurement_Table.is_anomaly, false).orderBy(Measurement_Table.is_failed, false).orderBy(Measurement_Table.id, true).queryList();
-		return measurements;
+		if (WebsitesSuite.NAME.equals(test_group_name))
+			return SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true)).orderBy(Measurement_Table.is_anomaly, false).orderBy(Measurement_Table.is_failed, false).orderBy(Measurement_Table.id, true).queryList();
```

Comment:
Line too long, please wrap! Also, please, add `{` and `}`.

Reply:
done